test/controllers/correctReturn/RemoveSmallProducerConfirmControllerSpec.scala

```diff
@@ -88,10 +88,26 @@ class RemoveSmallProducerConfirmControllerSpec extends SpecBase with MockitoSuga
           val result = route(application, request).value
 
           status(result) mustEqual SEE_OTHER
-          redirectLocation(result).value mustEqual routes.SmallProducerDetailsController.onPageLoad(mode).url
+          redirectLocation(result).value mustEqual routes.ExemptionsForSmallProducersController.onPageLoad(mode).url
         }
       }
 
+        s"must redirect to Small Producer Details when small producer not found for a GET in $mode" in {
```

Comment:
Update the name of this test to reference that it is when user answers contains at least one small producer